tests/Collector/SilverStripeCollectorTest.php

```diff
@@ -26,6 +27,12 @@ public function setUp()
         parent::setUp();
         DebugBar::initDebugBar();
         $this->collector = DebugBar::getDebugBar()->getCollector('silverstripe');
+        Requirements::set_backend(Requirements_Backend::create());
```

Comment:
This looks like a regression in 4.0 - would you mind PR-ing to core to re-add this via a test state class or maybe adding to the `sapphireteststate` class?